lms/static/sass/features/_course-experience.scss

```diff
@@ -1,3 +1,66 @@
+// Course call to action message
+.course-message {
+  .message-author {
+    display: inline-block;
+    width: $baseline*7/2;
```

Comment:
For example, if an Open edX instance doubles ``$baseline`` for some reason, would we want this message author area to double in size?